e2e/images_test.go

```diff
@@ -49,14 +32,14 @@ func TestImageList(t *testing.T) {
 		dir := fs.NewDir(t, "")
 		defer dir.Remove()
 
-		digest := insertBundles(t, cmd, dir, info)
+		insertBundles(t, cmd, info)
 
-		expected := `APP IMAGE                                                                                      APP NAME
+		expected := `APP IMAGE                     APP NAME
 %s push-pull
-a-simple-app:latest                                                                            simple
-b-simple-app:latest                                                                            simple
+a-simple-app:latest           simple
+b-simple-app:latest           simple
 `
-		expectedOutput := fmt.Sprintf(expected, info.registryAddress+"/c-myapp@"+digest)
```

Comment:
This test was useful, we want to test that images pulled by digest are visible in the `image ls` output.

Reply:
unfortunately the build / push behaviour is hardly impacted by this PR, and I was not able to keep the semantic this test relied on